tests/api_tests/test_questionnaire_response_api.py

```diff
@@ -574,8 +573,6 @@ def test_gror_consent_with_duplicate_answers(self, mock_logging):
 
         summary = self.send_get("Participant/%s/Summary" % participant_id)
         self.assertEqual(summary['consentForGenomicsROR'], 'SUBMITTED')
-        # Generic verification PDR generator logged a warning.  TODO:  Possible to test logged message w/regex?
```

Comment:
replace with check of the resource generator logging?

Reply:
How about in the next resource PR?

Reply:
That's fine.